packages/cli/commands/project/migrateApp.js

```diff
@@ -121,6 +125,7 @@ exports.handler = async options => {
     type: 'confirm',
     message: i18n(`${i18nKey}.createAppPrompt`),
   });
+  process.stdin.resume();
```

Comment:
See https://github.com/SBoudrias/Inquirer.js/issues/662

Reply:
Nice, good find. Does this work without the `process.stdin.setRawMode(true);` part? I saw one person mention needing that.

Reply:
It does work without `process.stdin.setRawMode(true);`.